img/app-dark.svg

```diff
@@ -1 +0,0 @@
-<svg xmlns="http://www.w3.org/2000/svg" height="16" viewBox="0 0 16 16" width="16" version="1.1"><path d="m6.94 0.5c-0.24 0-0.44 0.2-0.44 0.44v1.26c-0.5 0.14-1.05 0.35-1.53 0.65l-0.91-0.91c-0.16-0.18-0.46-0.19-0.62 0l-1.5 1.5c-0.18 0.17-0.18 0.46 0 0.62l0.91 0.91c-0.284 0.48-0.5 1-0.65 1.53h-1.26c-0.24 0-0.44 0.2-0.44 0.44v2.12c0 0.25 0.19 0.44 0.44 0.44h1.26c0.14 0.54 0.36 1.05 0.65 1.53l-0.91 0.91c-0.18 0.17-0.18 0.45 0 0.62l1.5 1.5c0.18 0.18 0.46 0.18 0.62 0l0.91-0.91c0.48 0.285 1 0.5 1.53 0.65v1.26c0 0.25 0.2 0.44 0.44 0.44h2.12c0.24 0 0.45-0.2 0.44-0.44v-1.26c0.54-0.14 1.05-0.36 1.53-0.65l0.91 0.91c0.17 0.18 0.45 0.18 0.62 0l1.5-1.5c0.18-0.17 0.18-0.45 0-0.62l-0.91-0.91c0.29-0.48 0.5-1 0.65-1.53h1.26c0.24 0 0.45-0.2 0.44-0.44v-2.12c0-0.24-0.2-0.44-0.44-0.44h-1.26c-0.14-0.54-0.36-1.05-0.65-1.53l0.91-0.91c0.18-0.17 0.18-0.45 0-0.62l-1.5-1.5c-0.17-0.18-0.45-0.18-0.62 0l-0.91 0.91c-0.48-0.29-1-0.5-1.53-0.65v-1.26c0-0.24-0.2-0.44-0.44-0.44h-2.12zm1.06 4a3.5 3.5 0 0 1 3.5 3.5 3.5 3.5 0 0 1 -3.5 3.5 3.5 3.5 0 0 1 -3.5 -3.5 3.5 3.5 0 0 1 3.5 -3.5z"/></svg>
```

Comment:
We want to keep this file, It's our temp icon for the dashboard.

Reply:
Fixed in last commit